db/migrate/20230222155438_create_likes.rb

```diff
@@ -0,0 +1,13 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :authorId
+      t.integer :postId
```

Comment:
- [ ] Please 🙏🏽 use references it will create the corresponding columns for author_id and post_id and the corresponding index. Just like in the comments table.